src/System.Private.CoreLib/shared/System/Text/Unicode/Utf8Utility.cs

```diff
@@ -45,16 +44,29 @@ public static unsafe int GetIndexOfFirstInvalidUtf8Sequence(ReadOnlySpan<byte> u
         }
 
 #if FEATURE_UTF8STRING
```

Comment:
why we have this ifdef now?

Reply:
The feature as a whole is still behind a feature flag. I'm trying not to add too many APIs outside of the feature flag if we can help it, since it allows us to back this whole thing out cleanly if we decide not to ship it.